### PR TITLE
Update OCP-22101 OCP-21090 OCP-21306 OCP-25353 into latest

### PR DESCRIPTION
**Pull request by tzhou5**
@yasun1 @xueli181114 @yuwang-RH Please help review the updates, thanks

### CODE REVIEW

File: lib/rules/web/ocm_console/cluster_detail.xyaml [hunk 1]
```diff
@@ -224,47 +224,47 @@ check_ocp_tab_in_detail_page:
...
         - selector:
             xpath: //p[text()='The cluster currently does not have any metrics data. Try again later.']
 check_ocp_details_in_detail_page:
     elements:
         - selector:
-            xpath: //h2[@class='pf-c-title card-title' and text()='Details']
+            xpath: //h2[text()='Details']
         - selector:
-            xpath: //dl[@class='cluster-details-item left']/dt[text()='Cluster ID']
+            xpath: //dl[@class='cluster-details-item']/dt[text()='Cluster ID']
```
Comment (xueli181114) [inline lib/rules/web/ocm_console/cluster_detail.xyaml]: Better not to use the class to locate. It's better to use //dl[1]/dt[]

File: lib/rules/web/ocm_console/cluster_detail.xyaml [hunk 2]
```diff
@@ -302,64 +304,66 @@ check_osd_tab_in_detail_page:
...
         - selector:
-            xpath: //dl[@class='cluster-details-item left']/dt[text()='Type']
+            xpath: //dl[@class='cluster-details-item']/dt[text()='Type']
         - selector:
-            xpath: //dl[@class='cluster-details-item left']/dd[text()='OSD']
+            xpath: //dl[@class='cluster-details-item']/dd/span[text()='OSD']
```
Comment (xueli181114) [inline lib/rules/web/ocm_console/cluster_detail.xyaml]: Better not to use the class to locate. It's better to use //dl[1]/dt[]
And should package the public part of OCP and OSD to an action

**Comment by xueli181114** [comment]
Others LGTM

File: lib/rules/web/ocm_console/cluster_detail.xyaml (hunk 1 above)
Comment (tzhou5) [reply lib/rules/web/ocm_console/cluster_detail.xyaml]: Updated

File: lib/rules/web/ocm_console/cluster_detail.xyaml (hunk 2 above)
Comment (tzhou5) [reply lib/rules/web/ocm_console/cluster_detail.xyaml]: Updated

**Pull request merged**